.github/workflows/build.yaml

```diff
@@ -65,22 +63,34 @@ jobs:
               - 'dataplane/**'
             containerfile:
               - 'build/Containerfile.dataplane'
-      - name: Setup QEMU
-        if: steps.filter.outputs.dataplane || steps.filter.outputs.containerfile
-        uses: docker/setup-qemu-action@29109295f81e9208d7d86ff1c6c12d2833863392 # v3.6.0
-      - name: Setup Docker Buildx
+      - name: Build container image for amd64
         if: steps.filter.outputs.dataplane || steps.filter.outputs.containerfile
-        id: buildx
-        uses: docker/setup-buildx-action@b5ca514318bd6ebac0fb2aedd5d36ec1b5c232a2 # v3.10.0
+        uses: docker/build-push-action@5cd11c3a4ced054e52742c5fd54dca954e0edd85 # v6.7.0
         with:
-          buildkitd-flags: "--debug"
-      - name: Build multi-arch container image
+          push: false
+          context: .
+          file: build/Containerfile.dataplane
+          tags: kong/blixt-dataplane:pr-${{ github.event.pull_request.number }}-amd64-${{ github.sha }}
+
+  docker-build-arm64:
+    name: docker-build-arm64
+    runs-on: ubuntu-22.04-arm
+    steps:
+      - name: Checkout
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - uses: dorny/paths-filter@v3
+        id: filter
+        with:
+          filters: |
+            dataplane:
+              - 'dataplane/**'
+            containerfile:
+              - 'build/Containerfile.dataplane'
```

Comment:
On further review I want to hold off on this extra dependency for now please, as I wanna do some looking around at what other kubernetes-sigs projects do for this.

```suggestion
```